packages/webcomponents/src/components/payment-form/payment-form.tsx

```diff
@@ -49,6 +49,22 @@ export class PaymentForm {
     this.submitButtonEnabled = false;
   }
 
+  showPaymentMethodTypeSelector() {
```

Comment:
Everything works correctly - one tiny suggestion: 

If you rewrite these as `getter` methods then they don't need to be written as functions to be called. You could do a very quick change:

```
  get PaymentMethodTypeSelector() {
    return this.card && this.bankAccount;
  }

  get SelectedPaymentMethodType() {
    if (this.PaymentMethodTypeSelector) {
      return this.selectedPaymentMethodType;
    } else if (this.card) {
      return PaymentMethodTypes.card;
    } else if (this.bankAccount) {
      return PaymentMethodTypes.bankAccount;
    } else {
      return PaymentMethodTypes.card;
    }
  }
```

And then when you are referencing this later you can simply reference it as `this.PaymentMethodTypeSelector` or `this.SelectedPaymentMethodType`

Reply:
Very nice suggestion however it leads to a naming conflict with a `@State` variable and resolving that will likely require a `setter`, so I am going to get this out as-is and we can always tweak that later (although I think we will be deprecating `PaymentForm` in the not-so-distant future)